src/server.rs

```diff
@@ -963,6 +963,7 @@ impl Server {
         if self.needs_cleanup {
             warn!("Server returned with session state altered, discarding state");
             self.query("DISCARD ALL").await?;
+            self.query("RESET ROLE;").await?;
```

Comment:
```suggestion
            self.query("DISCARD ALL; RESET ROLE;").await?;
```

Reply:
I dont believe we can chain queries

Reply:
You might be able to. We're just sending a `Q` message to the server. It's possible that two queries can be placed inside a single message.